### ISSUE TITLE
OpenDNS doesn't support DoT

### ISSUE THREAD

**Issue by bigdargon** [AdguardTeam/AdguardForiOS#1532]
Standard DNS:

```
https://doh.opendns.com/dns-query
```

FamilyShield (blocks adult content):

```
https://doh.familyshield.opendns.com/dns-query
```

More information: https://support.opendns.com/hc/en-us/articles/360038086532-Using-DNS-over-HTTPS-DoH-with-OpenDNS

https://support.opendns.com/hc/en-us/articles/360038463251-Querying-OpenDNS-using-DoH-for-developers-

```
Query format

OpenDNS follows RFC8484, accepting DoH (DNS over HTTPS) using both GET and POST methods containing queries in DNS Wire Format. Note that OpenDNS does not support queries in JSON format, nor DNS over TLS (DoT).
```

![7FFBA585-610B-438E-BAF7-B013DF997ED4](https://user-images.githubusercontent.com/10969626/77826226-b6fb7580-7140-11ea-9ba5-9c64c1ce0e2f.jpeg)
![B9AD3FA0-A9F2-4A06-9C07-DBCBFFCF7EEE](https://user-images.githubusercontent.com/10969626/77826234-bf53b080-7140-11ea-98a7-338543da079b.jpeg)

**Comment by bigdargon** [comment]
@SoLenny DoH has been lost in beta 4.0.1.  As mentioned above, OpenDNS supports DoH. More information: 

https://support.opendns.com/hc/en-us/articles/360038086532-Using-DNS-over-HTTPS-DoH-with-OpenDNS

https://support.opendns.com/hc/en-us/articles/360038463251-Querying-OpenDNS-using-DoH-for-developers-

**Comment by bigdargon** [comment]
.

**Comment by bigdargon** [comment]
fixed in v5.0.2 (509)
![AFEAA96A-51BD-472F-96E5-2A32E9AEF3E4](https://user-images.githubusercontent.com/10969626/89753477-a9002d00-db02-11ea-8793-50fb41ee652e.png)
![7023A77A-75E0-4B87-B47D-779392AF373F](https://user-images.githubusercontent.com/10969626/89753482-ae5d7780-db02-11ea-88fc-5c6f0f5ed99d.png)
